client-mvp-04/src/utils/uiKit/uiKit.js

```diff
@@ -33,117 +33,107 @@ const DevUiKit = () => {
 
   return (
     <div className="kit-container custom-scroll-bar">
-      <div className={'kit-title'}>VRMS DEV UI-KIT</div>
+      <div className="kit-title">VRMS DEV UI-KIT</div>
 
       {/*** HEADERS ***/}
       <div className="header-container">
-        <h1 className={'kit-sec-title'}>*** HEADERS ***</h1>
+        <h1 className="kit-sec-title">*** HEADERS ***</h1>
         <h1>VRMS h1</h1>
-        <p className={'dev-comment'}>h1 Header: 30px, bold; Main Page Header</p>
+        <p className="dev-comment">h1 Header: 30px, bold; Main Page Header</p>
         <h2>VRMS h2</h2>
-        <p className={'dev-comment'}>
+        <p className="dev-comment">
           h2 Header: 24px, semi-bold; Home Section Header
         </p>
-        <h2 className={'bold'}>VRMS h2</h2>
-        <p className={'dev-comment'}>
+        <h2 className="bold">VRMS h2</h2>
+        <p className="dev-comment">
           h2 Header: 24px, bold; Project Section Header / Onboarding Screens
         </p>
         <h3>VRMS h3</h3>
-        <p className={'dev-comment'}>
+        <p className="dev-comment">
           h3 Header: 18px, bold; Profile section header
         </p>
-        <h3 className={'regular'}>VRMS h3</h3>
-        <p className={'dev-comment'}>h3 Header: 18px, regular</p>
+        <h3 className="regular">VRMS h3</h3>
+        <p className="dev-comment">h3 Header: 18px, regular</p>
         <h4>VRMS h4</h4>
-        <p className={'dev-comment'}>h4 Header: 15px, bold</p>
+        <p className="dev-comment">h4 Header: 15px, bold</p>
         <h5>VRMS h5</h5>
-        <p className={'dev-comment'}>h5 Header: 14px, semi-bold</p>
+        <p className="dev-comment">h5 Header: 14px, semi-bold</p>
       </div>
 
       {/*** TEXT / LINKS ***/}
-      <div className={'link-container'}>
-        <h1 className={'kit-sec-title'}>*** TEXT / LINKS ***</h1>
+      <div className="link-container">
+        <h1 className="kit-sec-title">*** TEXT / LINKS ***</h1>
         <p>Text content</p>
-        <p className={'dev-comment'}>Main text: 'Open Sans', 14px, regular</p>
+        <p className="dev-comment">Main text: 'Open Sans', 14px, regular</p>
 
-        <p className={'italic'}>Text content</p>
-        <p className={'dev-comment'}>Main text: 'Open Sans', 14px, italic</p>
+        <p className="italic">Text content</p>
+        <p className="dev-comment">Main text: 'Open Sans', 14px, italic</p>
 
         <a key={'default-link'} href={'/'}>
```

Comment:
I noticed that there are {} around the a tag's attributes like key and href. In my experience these also do not need the {} around the strings. This also occurs in various places in the code for a tags. You may want to discuss this with a more senior member.

Reply:
Agreed.  You don't need the curly braces here.

Reply:
I agree.